spec/services/update_induction_tutor_reminder_spec.rb

```diff
@@ -16,18 +16,10 @@
     let(:method_name) { :remind_to_update_school_induction_tutor_details }
     subject { UpdateInductionTutorReminder.new(school) }
 
-    before do
-      allow(SchoolMailer).to(receive(method_name).and_call_original)
-    end
-
     it "calls SchoolMailer.remind_to_update_school_induction_tutor_details with the appropriate arguments" do
-      subject.send!
-
-      expect(SchoolMailer).to have_received(method_name).with(
-        school:,
-        sit_name: school.induction_tutor.full_name,
-        nomination_link: subject.instance_variable_get(:@nomination_link),
-      )
+      expect {
+        subject.send!
+      }.to have_enqueued_mail(SchoolMailer, method_name)
```

Comment:
removed those as couldn't use `instance_variable_get` open to suggestions on how to get this

Reply:
I've had a bit of a try, I expected [hash_including](https://www.rubydoc.info/gems/rspec-mocks/RSpec%2FMocks%2FArgumentMatchers:hash_including) to work but it wouldn't :( 

Think what you came up with is the best for the time being.